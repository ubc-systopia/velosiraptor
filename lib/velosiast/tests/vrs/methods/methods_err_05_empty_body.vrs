
// method with pre-condition
fn foo() -> bool
