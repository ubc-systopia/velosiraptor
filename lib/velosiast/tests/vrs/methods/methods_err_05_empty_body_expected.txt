error: method with no body must be declared extern, abstract or synth.
     --> tests/vrs/methods/methods_err_05_empty_body.vrs:3:1
      |
    3 |  fn foo() -> bool
      |  ^^ make this an `extern fn`, `abstract fn` or `synth fn`
error: could not compile due to 1 previous errors; 0 warnings emitted


@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
